cura/Machines/Models/IntentModel.py

```diff
@@ -74,21 +74,7 @@ def _update(self) -> None:
                     new_items.append(intent)
                     layer_heights_added.append(intent["layer_height"])
 
-        # Now that we added all intents that we found something for, ensure that we set add ticks (and layer_heights)
```

Comment:
This will break the recommended view (which is also mentioned in the comment that it's needed for that reason).

Reply:
Fixed in d99386cf2